lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -28,17 +28,24 @@ $(document).ready(function() {
 
   // Enable the Onboarding experiment.
   if (Drupal.settings.dsOnboarding.enabled && localStorage.getItem(localStorageKey) !== 'true') {
+    const slideComponents = {ContextSlide, InstructionSlide, ReportbackItemKudosSlide};
     const jsxElementHook = 'jsx-onboarding';
-    const slides = [InstructionSlide]; // ReportbackItemKudosSlide, ContextSlide
+    let slides = [];
 
-    $('.chrome').find('> .wrapper').before(`<div id="${jsxElementHook}"></div>`);
-    $('.messages:not(.error)').remove();
+    slides = Drupal.settings.dsOnboarding.slides.map(function (slide) {
```

Comment:
I know this is nit-picky, but I'm confused why you're setting `slides` to an empty array on line 33 and then immediately re-writing it on line 35 with the return value of `.map()`. Wouldn't it make more sense to remove the original empty array definition and just define it as the result of the map?

```js
const slides = Drupal.settings.dsOnboarding.slides.map(name => allSlides[name]);
```